light/store/db/db.go

```diff
@@ -238,7 +238,7 @@ func (s *dbs) Prune(size uint16) error {
 		append(s.lbKey(1<<63-1), byte(0x00)),
 	)
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
How’s panic better than returning ?

Reply:
It's just more consistent with how we handle iterator errors in other methods

Reply:
While more consistent do we want to panic on this error? Is it something that would cause a node failure?